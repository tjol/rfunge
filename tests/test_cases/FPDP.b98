"PDPF"4#v(1F2FD3FAQKP1F2FDEC2FYP0"57.0"RTP1F2FDEC2FYTG1FNYPaF5FYL01-FYP@
        >a"deliaf gnidaol"dk,@

 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ; not testing as thoroughly as FPSP as the two-cell values are a pain ;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;