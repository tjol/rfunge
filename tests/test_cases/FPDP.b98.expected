0.626381 0.750000 0.931596 1.333333 0.200000 